src/vs/workbench/contrib/positronConnections/browser/positronConnections.tsx

```diff
@@ -248,7 +248,7 @@ const PositronConnectionsItem = (props: React.PropsWithChildren<PositronConnecti
 				}
 			</div>
 			<div
-				className={`connections-icon codicon codicon-${icon}`}
+				className={`connections-icon codicon codicon-${icon} ${props.item.preview ? '' : 'disabled'}`}
```

Comment:
We have a helper for building class names called `positronClassNames`.

Example:

```
			<div
				className={positronClassNames(
					'sizer',
					{ 'resizing': resizing && props.showResizeIndicator }
				)}
				onPointerDown={pointerDownHandler}
			/>
```